### PR TITLE
Add crosssect documentation

### LINKED ISSUE #120: Specification documentation for crosssects
Add a specification to the eMach repository for how `crosssect` classes should be implemented. Please add this as a `README.md` file in `eMach\model_obj\crosssects\`. This file should contain the following:

- specification of the expected folder contents of a `crosssect` class's folder (list of expected files and their naming convention)
- specification of the drawing file(s) to include. I suggest: `svg` files (no PDFs) and if any source used to create the svg (i.e., tikZ). 
- specification of the contents of the drawing file and how it should be presented (italicized times new roman font, `x-y` coordinate system centered at drawing origin, etc.)
- specification of how `crosssect` class property names should correspond to the drawing file

### PR DESCRIPTION
**Pull request by ngadiyar93**
This PR adds documentation containing instructions to add new crosssects to eMach

### CODE REVIEW

**Review by Bharat-Ramadas — DISMISSED** [review]
Looks good @ngadiyar93 !

**Review by elsevers — CHANGES_REQUESTED** [review]
Please add missing periods and colons.

**Comment by ngadiyar93** [comment]
@elsevers, @Bharat-Ramadas @aimthiazz @ASHAD1  I have updated the markdown document. It can be reviewed.

**Review by elsevers — CHANGES_REQUESTED** [review]
Please add example of a cross-section drawing that is compliant with these guidelines

File: model_obj/crosssects/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Specifications for `Crosssect` class implementation
+
+The `Crosssect` class implements cross-sections of standard geometry shapes that are usually encountered when using eMach. This document specifies the rules to be followed to add a new cross-section to eMach.
```
Comment (Bharat-Ramadas) [inline model_obj/crosssects/README.md]: It might be better to say "... shapes that are usually encountered in machine design..." rather than "eMach"

Comment (ngadiyar93) [reply model_obj/crosssects/README.md]: @Bharat-Ramadas thank you!. Fixed this now.

**Review by Bharat-Ramadas — DISMISSED** [review]
Nice work!

**Review by aimthiazz — DISMISSED** [review]
Looks good.

**Review by aimthiazz — APPROVED** [review]
Looks good!!

**Review by elsevers — APPROVED** [review]

**Pull request merged**